lexoid/core/utils.py

```diff
@@ -46,8 +46,10 @@ def split_pdf(input_path: str, output_dir: str, pages_per_split: int):
 
 
 def create_sub_pdf(
-    input_path: str, output_path: str, page_nums: Optional[tuple] = None
+    input_path: str, output_path: str, page_nums: Optional[Union[tuple|int]] = None
 ) -> str:
+    if isinstance(page_nums, int):
+        page_nums = (page_nums,)
```

Comment:
If we do this then will it fail if the user accidentally does `page_nums=(2,)`?

Reply:
Why would it? `(2,)` is a `tuple`, right?

Reply:
Gets interpreted as `int`, I think. Can we add a test and verify?

Reply:
yes, make sense.